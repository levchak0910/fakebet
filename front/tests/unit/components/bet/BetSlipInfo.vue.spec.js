import {createWrapper, disableFile, DATA, mapProperties, changeDataToRenderableMode} from "../../__utils__";
import Component from "@/components/bet/betSlip/Info";

disableFile();

describe("bet/BetSlipInfo.vue", () => {

    let wrapper;
    let mountWrapper = (options = {}) => {
        wrapper = createWrapper(Component, {
            stubs: ["app-bet-slip-info-line"],
            ...options,
        });
    };

    it("Testing snapshot", () => {
        mountWrapper({
            props: {
                bets: changeDataToRenderableMode(DATA.betSlipBetsInfo),
                ...mapProperties("lineMode", "betSlipIndex"),
            },
        });

        expect(wrapper.element).toMatchSnapshot();
    });

});
