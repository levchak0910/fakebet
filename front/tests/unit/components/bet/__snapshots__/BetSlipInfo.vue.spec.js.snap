// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bet/BetSlipInfo.vue Testing snapshot 1`] = `
<div>
  <app-bet-slip-info-line-stub
    betindex="0"
    betslipindex="{{betSlipIndex}}"
    bookie="{{bookie 1}}"
    datetmpl="{{12.12}}"
    guest="{{guest team 1}}"
    home="{{home team 1}}"
    league="{{league 1}}"
    linemode="{{lineMode}}"
    matchresult="{{win}}"
    score="{{3 : 2}}"
    type="{{1}}"
  />
  <app-bet-slip-info-line-stub
    betindex="1"
    betslipindex="{{betSlipIndex}}"
    bookie="{{bookie 2}}"
    datetmpl="{{12.12}}"
    guest="{{guest team 2}}"
    home="{{home team 2}}"
    league="{{league 2}}"
    linemode="{{lineMode}}"
    matchresult="{{lose}}"
    score="{{1 : 1}}"
    type="{{2}}"
  />
</div>
`;
